lab2.js

```diff
@@ -118,8 +181,17 @@ assert((new Human()).sayHello(new Klingon()) === 'nuqneH',
 // will test your code)
 //*********************************************************
 
+var animals = ['bear', 'cat', 'dog', 'bee'];
+var foods = ['beets', 'soda', 'hotdog'];
 function lastLetterSort(stringArray) {
   function byLastLetter(a, b) {
+    if (a.charAt(a.length - 1) > b.charAt(b.length - 1)) {
+      return 1;
+    }
+    if (a.charAt(a.length - 1) < b.charAt(b.length - 1)) {
+      return -1;
+    }
+    return 0;
```

Comment:
can also return the difference between the two values. As long as sort gets a positive number, a negative number, or 0, it's happy.